script/publish-to-npm.js

```diff
@@ -118,6 +118,15 @@ new Promise((resolve, reject) => {
       } else {
         npmTag = `nightly-${currentBranch}`
       }
+
+      const currentJson = JSON.stringify(fs.readFileSync(path.join(tempDir, 'package.json'), 'utf8'))
+      currentJson.name = 'electron-nightly'
+      rootPackageJson.name = 'electron-nightly'
+
+      fs.writeFileSync(
+        path.join(tempDir, 'package.json'),
+        JSON.stringify(currentJson, null, 2)
+      )
```

Comment:
i don't suppose we could arrange to have this information the _first_ time we write to `tempDir/package.json`?

Reply:
Unfortunately not, it's a chicken --> egg problem.  We need the temporary package.json to get the release, and we need to the release to get whether it's nightly or not.

In an ideal world this would all be refactored, but with no tests for this file I'd like to keep the changes to a minimum to reduce the chance this breaks at the end of a 2 hour release 😆